### PR TITLE
Updating homepage links

### PR DESCRIPTION
**Pull request by Charlie-Henry**
https://github.com/cityofaustin/atd-data-tech/issues/15954

- Removed the SD 2023 link (no longer a thing the city tracks)
- Fixed open data portal link
- Added link to traffic incidents page
- Updated image for signal monitor 

https://deploy-preview-417--austin-transportation.netlify.app/

### CODE REVIEW

**Comment by Charlie-Henry** [comment]
Screenshot 2024-02-23 at 4 32 13 PM

too much [wide Putin](https://www.youtube.com/watch?v=ZLcgsBAXNaQ)?

File: public/assets/traffic-incidents.png
Comment (chiaberry) [inline public/assets/traffic-incidents.png:1]: I do think the text is too wide/stretched out

**Review by chiaberry — APPROVED** [review]
I looked at the traffic incidents image again in context, on the netlify preview, and while I still think its stretched out, its not terrible when seen at that size.

**Review by mddilley — COMMENTED** [review]
Wondering if cropping the traffic incidents image to the map below the blue banner would make this less stretched out? If seeing the whole page is important, the full map in the bottom part could be cutoff too. Either way, the code changes look great!

**Comment by johnclary** [comment]
i don't like the image. it looks janky. sorry @Charlie-Henry! the code is 💯

**Comment by johnclary** [comment]
cc @amenity

**Comment by amenity** [comment]
This one looks good!
Screenshot 2024-02-28 at 2 06 31 PM

FWIW, [here are my instructions for tile images](https://atd-dts.gitbook.io/wiki/product-ops/index-issue-specifications#index-issue-images). I think that if you can zoom in on a cool part of the UI, that's fine. Otherwise screenshots don't work for tiles.

**Pull request merged**